test/IdentityServer4.RavenDB.Storage.Tests/IntegrationTestBase.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using IdentityServer4.RavenDB.Storage.DocumentStoreHolder;
+using IdentityServer4.RavenDB.Storage.Options;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.TestDriver;
+
+namespace IdentityServer4.RavenDB.Storage.Tests
+{
+    public class IntegrationTestBase : RavenTestDriver
+    {
+        internal ConfigurationDocumentStoreHolder GetConfigurationDocumentStoreHolder()
```

Comment:
What's the purpose of making these methods `internal`? Can we use `protected` instead?

Reply:
Since `OperationalDocumentStoreHolder` and `ConfigurationalDocumentStoreHolder` are internal classes - we don't want them to be available to client code - you can't mark these methods as `protected` due to inconsistent accessibility.

Reply:
That's right 👍